engine/src/main/scala/cromwell/logging/TerminalLayout.scala

```diff
@@ -22,25 +24,23 @@ object TerminalLayout {
   implicit class ColorString(msg: String) {
     def colorizeUuids: String = {
       "UUID\\((.*?)\\)".r.findAllMatchIn(msg).foldLeft(msg) {
-        case (l, r) =>
-          val color = if (Option(System.getProperty("RAINBOW_UUID")).isDefined)
-            Math.abs(17 * r.group(1).substring(0,8).map(_.toInt).product) % 209 + 22
-          else 2
```

Comment:
I liked it when it used to be there, but we shut off colors in server logging. Maybe useful in tests sometime in the future?

Basically, it generated a different color for each UUID. When viewing a crap ton of workflows going by, the different UUIDs would be clearly highlighted with different colors.

But yes, the code is vestigial at the moment. RIP